taccsite_cms/static/site_cms/css/src/_imports/generics/fonts.css

```diff
@@ -29,6 +29,14 @@ Styleguide Generics.Fonts
   font-display: swap;
 }
 
```

Comment:
Self, please remove this line, to show the relationship of Regular to RegularItalic.

```suggestion
```